crates/swc/Cargo.toml

```diff
@@ -53,7 +53,7 @@ serde = {version = "1", features = ["derive"]}
 serde_json = "1"
 sourcemap = "6"
 swc_atoms = {version = "0.2", path = "../swc_atoms"}
-swc_common = {version = "0.16.0", path = "../swc_common", features = ["sourcemap", "concurrent"]}
+swc_common = {version = "0.16.0", path = "../swc_common", features = ["sourcemap", "concurrent", "plugin-rt"]}
```

Comment:
`plugin-rt` should be enalbed via a cargo feature in the `node` crate.

`node/plugin` -> `swc/plugin` -> `swc_common/plugin-rt`.

Reply:
I think this is leftover while I move things around.